Description: install metainfo file to /usr/share/metainfo/ instead of /usr/share/appdata/
Author: Vincent Legout <vincent@legout.info>

Index: qalculate-gtk/data/Makefile.am
===================================================================
--- qalculate-gtk.orig/data/Makefile.am
+++ qalculate-gtk/data/Makefile.am
@@ -18,7 +18,7 @@ desktopdir = $(datadir)/applications
 
 appdata_IN_FILES = qalculate-gtk.appdata.xml.in
 appdata_DATA = $(appdata_IN_FILES:.xml.in=.xml)
-appdatadir = $(datadir)/appdata
+appdatadir = $(datadir)/metainfo
 @INTLTOOL_XML_RULE@
 
 pixmapsdir = $(datadir)/pixmaps
